microk8s-resources/wrappers/run-kubelite-with-args

```diff
@@ -153,6 +162,15 @@ else
   echo "\`/var/lib/kubelet\` already exists. CSI add-ons have to point to $SNAP_COMMON for kubelet."
 fi
 
+# Sanatize arguments for kubelet
+if [ -e $SNAP_DATA/var/lock/no-arg-sanitisation ] 
+then 
+  echo "Skipping argument sanitisation for kubelet"
+else
+  sanatize_args_kubelet
+fi
+
+
```

Comment:
We need some argument hadling for the remaining two services (controller manager and scheduler) too.

Reply:
Where are we initializing them in kubelite?

Reply:
Got it.